Lib/pathlib.py

```diff
@@ -1253,6 +1253,15 @@ def readlink(self):
         Return the path to which the symbolic link points.
         """
         self._unsupported("readlink")
+    readlink._unsupported = True
```

Comment:
As far as I can tell ``readlink._unsupported = True`` is never set to ``False``? Perhaps this could also switch to ``readlink._supported = False``, the double- and triple-negatives are a bit confusing at times.

Reply:
It would be unset if a user subclasses `PathBase` and adds their own `readlink()` method. I've added a comment and renamed it to `supported` - does it make more sense now?

Reply:
I was confused as to why ``Path`` didn't change this -- but of course ``os.path.realpath`` is used instead!

A